modules/ROOT/pages/create-map-json-xml-to-outbound-x12.adoc

```diff
@@ -37,36 +37,37 @@ The template has the X12 configuration for the below X12 transaction set:
 
 * Version +
 `4010`
-* Transaction Set + 
+* Transaction Set +
 `850 (Purchase order)`
 
 Follow these steps to set the target X12 message type:
 
 . Click the *Global Elements* tab at the base of the Studio canvas in `global-configs.xml`.
 . From *Global Configuration Elements*, select *X12 EDO Config* and click *Edit*.
```

Comment:
Is it really X12 EDO? Just asking because I've only seen X12 EDI

Reply:
Good catch